tests/visual/components/product/product.scss

```diff
@@ -0,0 +1,2 @@
+
+@import 'dist/components/product/product';
```

Comment:
This file `product.scss` should be prepended with an underscore, like `_product.scss`.

Because there is no underscore, that is why a `product.css` file is being compiled.